INDEX.md

```diff
@@ -0,0 +1,1436 @@
+# Free For Developers
```

Comment:
Why rename this file? I really would prefer not to, history etc matters

Reply:
Ya but mobile browsers just hang while this large readme gets opened.

Reply:
How does renaming it help? Sorry - for me on my mobiles this always just works

Reply:
The index file will not be opened directly. This can help the mobile users whose mobiles hang.

Reply:
I would rather we dont rename this file

Reply:
Ok I'll rename this back to README.md and current Readme to About.md

Reply:
Think the readme content can stay exactly as is

Reply:
So I'll just rename this right ? @ripienaar

Reply:
Leave readme as it was before basically

Reply:
:+1: